Tasks/HelmDeployV0/task.json

```diff
@@ -434,6 +434,14 @@
             "helpMarkDown": "If this is true, this task will fail if any errors are written to the error pipeline, or if any data is written to the Standard Error stream. Otherwise the task will rely on the exit code to determine failure.",
             "groupName": "advanced"
         },
+        {
+            "name": "publishMetadataFromTask",
+            "type": "boolean",
+            "label": "Publish metadata from task",
```

Comment:
publish pipeline metadata 

remove `fromTask`

Reply:
You mean the label or just the variable name?

Reply:
Both